### ISSUE TITLE
Bots dont function normal

### ISSUE THREAD

**Issue by CirovicFG** [yapb/yapb#497]
After updating to newest version to reduce their headshot skill, bots are so slow and stupid, they cant aim normally at enemies and they need around 1-6 seconds to shoot if you want i will give you IP to check them. they act really weird..

YAPB.CFG -> https://pastebin.com/FeXyZ5PJ
DIFFICULTY.CFG -> https://pastebin.com/4z9trug4

@jeefo please we need urgent help.

**Comment by CirovicFG** [comment]
**87.98.156.203:27018 you can join to check them, we need urgent fix for this. @jeefo**

**Comment by CirovicFG** [comment]
Here is video: https://streamable.com/kee2ey

**Comment by jeefo** [comment]
are you sure configs that you provided applied on server?

**Comment by CirovicFG** [comment]
Yes 100% only thing that MAYBE causes problem is that new bots are in folder yapbnew and old one are still in folder yapb maybe is that problem, but it shouldn't be because they are turned off.



![Screenshot_5](https://github.com/yapb/yapb/assets/101999668/12c7abeb-29a7-4b98-9e42-1a6d7b8f7fb5)
![Screenshot_6](https://github.com/yapb/yapb/assets/101999668/8707447e-c0d7-4d79-8f09-541901699766)


[](url)

**Comment by CirovicFG** [comment]
![Screenshot_7](https://github.com/yapb/yapb/assets/101999668/c98e0718-fe86-4456-ac8d-3fce855a59d2)

**Comment by jeefo** [comment]
> ![Screenshot_7](https://private-user-images.githubusercontent.com/101999668/285620877-c98e0718-fe86-4456-ac8d-3fce855a59d2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDA5NTU5NDYsIm5iZiI6MTcwMDk1NTY0NiwicGF0aCI6Ii8xMDE5OTk2NjgvMjg1NjIwODc3LWM5OGUwNzE4LWZlODYtNDQ1Ni1hYzhkLTNmY2U4NTVhNTlkMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTEyNVQyMzQwNDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMjdkZDFmNDU0ZTMzNWRmZGM0NGM5ZDM3MTZmOTdjYTgyMDg0YTM0OGNlNDJhYTI3NTdlMTQ3ZGQ2NjU0YTNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.wfpbyS23jThUJTFAq8Kl0i3-pvWuLvtztGAksh816bE)

well, you can,t install bot into a different folder, 5hey will use configs from the yapb, not yapbbew one.

and so, difficultty.cfg looks weird, try standard https://raw.githubusercontent.com/yapb/yapb/master/cfg/addons/yapb/conf/difficulty.cfg from the repo

**Comment by CirovicFG** [comment]
Okay testing in 2 minutes.

**Comment by CirovicFG** [comment]
Its working now good @jeefo, but headshot is still a thing and they movement kinda suck still (i use default difficulty config)

Noob = 0.2, 0.19, 7, 30, 30, 38, 30.0, 30.0, 40.0
Easy = 0.17, 0.16, 5, 40, 40, 32, 15.0, 15.0, 24.0
Normal = 0.15, 0.14, 10, 60, 60, 26, 5.0, 5.0, 10.0
Hard = 0.15, 0.13, 20, 60, 60, 23, 0.0, 0.0, 0.0
Expert = 0.1, 0.12, 30, 70, 70, 21, 0.0, 0.0, 0.0


will this reduce their headshot?

**Comment by jeefo** [comment]
set yb_difficulty to 3, set ``Hard = 0.15, 0.14, 10, 60, 60, 26, 5.0, 5.0, 10.0`` in difficulty.cfg, should be like normal.

Version 4.4. is not defaulting to expert with difficulty. 4 - is extreme difficulty niw,

**Comment by CirovicFG** [comment]
i did it, its good now. but their movement is really botish, they are so confused and not smart enough, anything i can to do fix them?
